src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
Classes should not begin with an empty line. Please remove the empty line at the start of the class definition.